easybuild/easyconfigs/l/libplinkio/libplinkio-0.9.8-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,33 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'CMakeMake'
+
+name = 'libplinkio'
+version = '0.9.8'
+
+homepage = "https://github.com/mfranberg/libplinkio"
+description = """A small C and Python library for reading PLINK genotype files."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'cstd': 'gnu17', 'noopt': True, 'extra_cflags': '-w'}
+
+github_account = "mfranberg"
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['de54dd4789b2da9b937a1f445dc6976153211dc35b4376a4cf561ad2ee861075']
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('binutils', '2.34'),
+]
+
+parallel = 1
```

Comment:
There seems to be no need to enforce sequential compilation, so please remove this line?